BioCommons/src/main/java/uk/ac/ebi/biostd/commons/files/MagicFolderUtil.java

```diff
@@ -57,6 +61,10 @@ public Path getGroupMagicFolderPath(long groupId, String secret) {
         return getMagicFolderPath(groupId, secret, GROUP_FOLDER_PREFIX);
     }
 
+    private Path getMagicFolderRelativePath(long id, String secret, String separator) {
+        return Paths.get(format("%s/%s-%s%d", secret.substring(0, 2), secret.substring(2), separator, id));
```

Comment:
We're doing this twice in this class. Perhaps we should reuse this method in `getMagicFolderPath`.